### ISSUE TITLE
[BUG] Weather reports are not populating with data since dec 2021

### ISSUE THREAD

**Issue by spydmobile** [intellifire/2022_season#9]
**Describe the bug**
Weather reports are not populating with data since dec 2021

**Things to do**

- [x] Find all the containers running the mapgenerator and list them here
- [x] Find the missing weather and identify where it can be found.
- [x] resolve the issue moving forward.

**Comment by spydmobile** [comment]
Found an instance on portainer 120 (test1)
This is writing the reports empty to the shared folders.
It is also generating reports from weather FILES not mongoDB.

The weather *IS* in mongo just not in the old system.
I think that the collection VM we shut down was the missing link - not sure?

**Comment by spydmobile** [comment]
https://portainer.120.intellifirenwt.com/#!/1/docker/containers

**Comment by spydmobile** [comment]
ok, this is a bug related to a feature. the feature was meant to not waste time collecting weather that has no data. @cfunk3 I have found this bug for real now. 

on jan 1, it started collecting weather, but found that there was no existing 2022 weather, so did not collect. it has been stuck like that ever since. 

Today I changed some code:

In repo intellifire_lrgs_collector (sparcs2021 branch) in the gardenlib.js file:
on line 2050 I changed:
```JS
 if (mostRecentRecord==false) {
 ```
 to
 ```JS
  if (mostRecentRecord=='falsed') {
  ```
  
  this means that this check for existing data is defeated and will allow weather to collect even though there is no data.
  
  This has kicks tarted weather collection as of now. So SPARCS2021 version of weather is missing all data from Jan 1 till today. This data exists in the intellifire weatehr database, just not in hte SPARCS 2021 system (Which still uses files)
